docs/contribution/community-components.md

```diff
@@ -0,0 +1,299 @@
+# Community components
+
+### What are community components?
+
+Community Components is our new library of custom components contributed by you.
+It is based off of our new contribution model.
+
+- **Development environment:** Unlike our old "add-on" model, community
+  components will no longer live in individual, per-team repos. Instead, they
+  will all live in the same `carbon-community-components` monorepo. This allows
+  us to provide a consistent API and a single workspace that is the source of
+  truth for community components, not just for development purposes but also for
+  consuming purposes. By having community components live in the same package,
+  we will be able to better surface your contributions.
+
+- **Website documentation:** Unlike before, when "add-ons" weren't really
+  surfaced in any of our documentation, community components will have a
+  gallery, coupled with component specific documentation, on our website. This
+  will allow the rest of the community to find and consume custom components
+  easier and thus reduce design and development effort. We also want to
+  encourage community members to contribute back, not just to Carbon, but to the
+  community components they're using as well. Just because core Carbon won't be
+  maintaining these, doesn't mean you can't contribute to them!
+
+- **Quality control:** We want to make contributing easy, but want to encourage
+  improvement, maintenance and contribution. We've created a recommended
+  checklist of comprehensive design and development guidelines. While there is
+  no requirement to meet any or all of these guidelines, component quality will
+  be displayed based off of how many guidelines you meet. Evaluation of level of
+  completeness is based on your assessment.
+
+  _If we're not required to meet any of the guidelines, why would we want to
+  create more work for ourselves by trying to meet most of them?_
+
+  Good question. Carbon will be endorsing components that meet 80% or more of
+  the guidelines as being on the same design and development level as core
+  Carbon components. This means that we will prioritize them when we surface
+  these components on our website. It is our goal that by endorsing these and
+  surfacing them more notably, more users will use and contribute to your
+  component and make it easier for you to maintain it.
+
+- **Maintainers:** The core Carbon team will not be responsible for maintaining
+  community components. The responsibility to maintain them will fall on the
+  team who contributed the component.
+
+<img src='./images/contribution_repo_diagram.png' alt='community components model diagram'/>
+
+The diagram above illustrates one way we could implement this vision for
+community contribution. In this model we would need both a Core Carbon and a
+Community monorepo. The Community repo would consume packages from Core Carbon
+repo and both monorepos would need to run on the same spec model (this has not
```

Comment:
if they are running the same spec across repos but community components that are variations on an existing component would require either duplicate specs OR the core specs would need to be updated to allow the flexibility.  (see previous comment about how variants in the separate repo would also be troublesome/non-DRY)